### PR TITLE
Fix issue (?) found by codespell

### PR DESCRIPTION
**Pull request by DimitriPapadopoulos**
codespell complains:
```
./NOTES:60: delink ==> unlink
```

### CODE REVIEW

**Comment by effigies** [comment]
Reading this, I believe it means that the file should still exist, but shouldn't be a hardlink or symlink to another file. I think "unlink" was specifically avoided for this reason.

**Comment by DimitriPapadopoulos** [comment]
I see. Should this perhaps be modified to something like the following?
_make sure _version.py is not a symlink or hardlink before changing it_

**Comment by effigies** [comment]
Would codespell accept quotes around it, like `"delink"`? Or what about "break the link"?

**Comment by DimitriPapadopoulos** [comment]
I think codespell doesn't  take punctuation into account. In any case don't bother about codespel, I can add a config file to shut it up if needed (unfortunately [`tox.ini` won't work so it'll be `setup.cfg` or `.codespellrc`](https://github.com/codespell-project/codespell#using-a-config-file)).

File: NOTES [hunk 1]
```diff
@@ -57,7 +57,7 @@
         - good to know: make_release_tree() creates hardlinks if possible. If
           'setup.py build' does this too, that would explain why it's not
           always necessary to rebuild when source files are changed.
-        - need to delink _version.py before changing it
+        - need to unlink _version.py before changing it
```
Comment (effigies) [inline NOTES:60]:
```suggestion
        - need to ensure _version.py is not a link before changing it
```

**Comment by effigies** [comment]
Sure. Made a suggestion.

**Comment by DimitriPapadopoulos** [comment]
Perfect! Thank you.

**Pull request merged**